### PR TITLE
add tezos-plasma-aggregator

### LINKED ISSUE #22: Create plasma-aggregator-tezos

### PR DESCRIPTION
**Pull request by syuhei176**
Close #22

### CODE REVIEW

File: packages/plasma-aggregator/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const instantiate = async (): Promise<Aggregator> => {
+  const kvs = new InMemoryKeyValueStore(Bytes.fromString('aaaaa'))
+  await kvs.open()
+  const network = process.env.TEZOS_NETWORK || 'babylonnet'
+  const wallet = new TzWallet(
+    await TezosWalletUtil.restoreIdentityWithSecretKey(process.env
+      .AGGREGATOR_PRIVATE_KEY as string),
+    {
+      url: process.env.MAIN_CHAIN_HOST as string,
+      apiKey: 'hooman',
```
Comment (daiki44) [inline packages/plasma-aggregator/src/index.ts]: I guess`apiKey` should be in `.env`

File: packages/plasma-aggregator/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const blockDb = await kvs.bucket(Bytes.fromString('block'))
+  const stateManager = new StateManager(stateDb)
+  const blockManager = new BlockManager(blockDb)
+  const witnessDb = await kvs.bucket(Bytes.fromString('witness'))
+  const eventDb = await kvs.bucket(Bytes.fromString('event'))
+  function depositContractFactory(address: Address) {
+    return new DepositContract(address, eventDb, wallet)
+  }
+  function commitmentContractFactory(address: Address) {
+    return new CommitmentContract(wallet.getConnection(address))
```
Comment (daiki44) [inline packages/plasma-aggregator/src/index.ts]: TODO: must update after #19

File: packages/plasma-aggregator/src/index.ts (hunk 1 above)
Comment (syuhei176) [reply packages/plasma-aggregator/src/index.ts]: fixed )b

File: packages/plasma-aggregator/src/index.ts (hunk 2 above)
Comment (syuhei176) [reply packages/plasma-aggregator/src/index.ts]: yes!

**Pull request merged**